homeassistant/util/yaml.py

```diff
@@ -16,6 +16,7 @@
 _LOGGER = logging.getLogger(__name__)
 _SECRET_NAMESPACE = 'homeassistant'
 _SECRET_YAML = 'secrets.yaml'
+_SECRET_CACHE = {}  # type: Dict
```

Comment:
Clear is probably a better option then you can execute it from bootstrap after the config is loaded. I think if you add double underscore the unit tests will be a challenge.